src/components/base/outline-google-map/outline-google-map.stories.ts

```diff
@@ -0,0 +1,202 @@
+      options: gestureOptions,
+      control: {
+        type: 'select',
+      },
+    },
+    disableDefaultUI: {
+      description: '**Boolean**: Clears the map controls.',
+    },
+    zoomControl: {
+      description:
+        '**Boolean**: Individually adds zoom controls to map after disabling controls.',
+    },
+    mapTypeControl: {
+      description:
+        '**Boolean**: Individually adds map type controls to map after disabling controls.',
+    },
+    scaleControl: {
+      description:
+        '**Boolean**: Individually adds scale controls to map after disabling controls.',
+    },
+    streetViewControl: {
+      description:
+        '**Boolean**: Individually adds street view controls to map after disabling controls.',
+    },
+    rotateControl: {
+      description:
+        '**Boolean**: Individually adds rotation controls to map after disabling controls.',
+    },
+    fullScreenControl: {
+      description:
+        '**Boolean**: Individually adds full screen controls to map after disabling controls.',
+    },
+  },
+  parameters: {
+    viewMode: 'docs',
+    previewTabs: {
+      canvas: {
+        hidden: false,
+      },
+    },
+    docs: {
+      description: {
+        component: `This component renders a Google Map.`,
+      },
+      source: {
+        code: `
+<outline-google-map
+  api-key="{{ apiKey }}"
+  map-type="{{ mapTypeId }}"
+  center-latitude="{{ mapStartingLat }}"
+  center-longitude="{{ mapStartingLong }}"
+  fit-to-markers
+  disable-ui
+>
+  <outline-map-marker
+    slot="markers" 
+    latitude="{{ locationLat }}" 
+    longitude="{{ locationLong }}"
+    icon="{{ pngUrlorOutlineSVG }}"
+    iconFillColor="{{ htmlColororOutlineVar }}">
+      {{ mapMarkerDefaultSlot }}
+  </outline-map-marker>
+</outline-google-map>
+        `,
+      },
+    },
+  },
+};
+
+const Template = ({
+  apiKey,
+  zoom,
+  mapType,
+  centerLatitude,
+  centerLongitude,
+  fitToMarkers,
+  disableDefaultUI,
+  zoomControl,
+  mapTypeControl,
+  scaleControl,
+  streetViewControl,
+  rotateControl,
+  fullScreenControl,
+  gestureHandling,
+  defaultSlot,
+}): TemplateResult => {
+  zoom = zoom > 0 ? zoom : null;
+  return html`
+      <outline-google-map
+        apiKey="${apiKey}"
+        .zoom="${zoom}"
+        mapType="${mapType}"
+        centerLatitude="${ifDefined(centerLatitude)}"
+        centerLongitude="${ifDefined(centerLongitude)}"
+        gestureHandling="${gestureHandling}"
+        ?fitToMarkers="${fitToMarkers}"
+        ?disableDefaultUI="${disableDefaultUI}"
+        ?zoomControl="${zoomControl}"
+        ?mapTypeControl="${mapTypeControl}"
+        ?scaleControl="${scaleControl}"
+        ?streetViewControl="${streetViewControl}"
+        ?rotateControl="${rotateControl}"
+        ?fullScreenControl="${fullScreenControl}"
+      >
+      ${unsafeHTML(`${defaultSlot}`)}
+      </outline-google-map>
+    `};
+
+export const OutlineGoogleMap = Template.bind({});
+OutlineGoogleMap.args = {
```

Comment:
Minor: I usually define defaults in the configuration `args` property. That way if we need to make another story later, we don't need to redefine everything, just the changes.